aptos-move/framework/move-stdlib/sources/vector.move

```diff
@@ -211,6 +211,21 @@ module std::vector {
         pragma intrinsic = true;
     }
 
+    /// Push a new element to the back of the vector if it is not already in the vector.
+    /// This is O(n) in the worst case and preserves ordering of elements in the vector.
+    /// Return `true` if `key` did not already exist in the vector and `false` otherwise.
+    public fun push_unique<Element: drop>(v: &mut vector<Element>, e: Element): bool {
+        if (contains(v, &e)) {
+            false
+        } else {
+            push_back(v, e);
+            true
+        }
+    }
+    spec push_unique {
```

Comment:
No need to use this. This is only for functions which have builtin treatment in the prover, this is just a regular Move function. Same below.